packages/critters/README.md

```diff
@@ -87,32 +89,118 @@ All optional. Pass them to `new Critters({ ... })`.
+  - `true` to inline critical font-face rules and preload the fonts
+  - `false` to don't inline any font-face rules and don't preload fonts
+- `keyframes` **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Controls which keyframes rules are inlined.\* Values:
+  - `"critical"`: _(default)_ inline keyframes rules used by the critical CSS
+  - `"all"` inline all keyframes rules
+  - `"none"` remove all keyframes rules
+- `compress` **[Boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** Compress resulting critical CSS _(default: `true`)_
+- `logLevel` **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Controls [log level](#loglevel) of the plugin _(default: `"info"`)_
+- `logger` **[object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** Provide a custom logger interface [logger](#logger)
+- `includeSelectors` **[RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)** | **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Provide a list of selectors that should be included in the critical CSS. Accepts both RegExp and string.
+
+### Include/exclude rules
+
+We can include or exclude rules to be part of critical CSS by adding comments in the CSS
+
+Single line comments to include/exclude the next CSS rule
+
+```css
+/* critters:exclude */
+.selector1 {
+  /* this rule will be excluded from critical CSS */
+}
+
+.selector2 {
+  /* this will be evaluated normally */
+}
+
+/* critters:include */
+.selector3 {
+  /* this rule will be included in the critical CSS */
+}
+
+.selector4 {
+  /* this will be evaluated normally */
+}
+```
+
+Including/Excluding multiple rules by adding start and end markers
+
+```css
+/* critters:exclude start */
+
+.selector1 {
+  /* this rule will be excluded from critical CSS */
+}
+
+.selector2 {
+  /* this rule will be excluded from critical CSS */
+}
+
+/* critters:exclude end */
+```
+
+```css
+/* critters:include start */
+
+.selector3 {
+  /* this rule will be included in the critical CSS */
+}
+
+.selector4 {
+  /* this rule will be included in the critical CSS */
+}
+
+/* critters:include end */
+```
+
+### Critters container
+
+By default Critters evaluates the CSS against the entire input HTML. Critters evaluates the Critical CSS by reconstructing the entire DOM and evaluating the CSS selectors to find matching nodes. Usually this works well as Critters is lightweight and fast.
+
+For some cases the input HTML can be very large or deeply nested which makes the reconstructed DOM much larger which in turn can slow down the critical CSS generation. Critters is not aware of viewport size and what specific nodes are above the fold since there is not headless browser involved.
```

Comment:
```suggestion
For some cases, the input HTML can be very large or deeply nested which makes the reconstructed DOM much larger, which in turn can slow down the critical CSS generation. Critters is not aware of viewport size and what specific nodes are above the fold since there is not a headless browser involved.
```